runhouse/resources/secrets/secret.py

```diff
@@ -231,14 +245,28 @@ def _delete_local_config(self):
         if os.path.exists(config_path):
             os.remove(config_path)
 
-    def _delete_vault_config(self, headers: str = rns_client.request_headers):
+    def _delete_secret_configs(self, headers: Optional[str] = None):
+        headers = headers or rns_client.request_headers
+
+        # Delete secrets in Vault
+        resource_uri = rns_client.resource_uri(self.rns_address)
         resp = requests.delete(
-            f"{rns_client.api_server_url}/{self.USER_ENDPOINT}/{self.name}",
+            f"{rns_client.api_server_url}/user/secret/{resource_uri}",
```

Comment:
should we keep `self.USER_ENDPOINT` here?

Reply:
and should we talk about group endpoint at some point or not a priority for now?

Reply:
I think out of scope for this PR, but we should def talk about it for next steps